Content.Server/_Goobstation/StationEvents/Components/GameDirectorComponent.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Content.Server._DV.Administration.Commands; // DeltaV - Easier Debugging
+using Content.Server._Goobstation.StationEvents.Metric;
+using Content.Shared._Goobstation.StationEvents.Metric; // DeltaV - shared the prototypes for our tests
+using Content.Shared._Goobstation.StationEvents; // DeltaV - shared the prototypes for our tests
+using Content.Shared.EntityTable.EntitySelectors;
+using Robust.Shared.Prototypes;
```

Comment:
Remove unused using directives
```suggestion
using Content.Server._DV.Administration.Commands; // DeltaV - Easier Debugging
using Content.Shared._Goobstation.StationEvents.Metric; // DeltaV - shared the prototypes for our tests
using Content.Shared._Goobstation.StationEvents; // DeltaV - shared the prototypes for our tests
using Robust.Shared.Prototypes;
```